includes/rest-api/class-sensei-rest-api-course-users-controller.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * File contains the Sensei_REST_API_Course_Users_Controller class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Course users actions controller class.
+ *
+ * @since x.x.x
+ */
+class Sensei_REST_API_Course_Users_Controller extends \WP_REST_Controller {
```

Comment:
What do you think about replace `users` to `students` to follow properly our domain name?